README.md

```diff
@@ -1 +1,138 @@
-# school_library
\ No newline at end of file
+# Oop-School-library
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)

```

Comment:
![image](https://github.com/Zuwairanajma/school_library/assets/93039359/3791b1f4-a540-4a23-a0bc-cb2c75897503)

- [ ] It would be nice if you fix the highlighted link in the image above because right now it is broken 😨 Kindly fix it or remove it `(if you don't have any dedicated section for the broken link)` to make your readme descriptive and professional 👍